server/routes/snippets.js

```diff
@@ -95,57 +84,13 @@ router.post('/', function (req, res) {
   });
 });
 
-/* GET snippet's rating from user*/
-router.get('/:snippet_id/users/:user_id', function (req, res) {
-  models.Rating.findAll({ where: { UserId: req.params.user_id, SnippetId: req.params.snippet_id } }).then(function (ratings){
-    var mappedRatings = ratings.map(function (rating) {
-      return rating.toJson();
-    });
-
-    res.send(mappedRatings);
-  });
-});
-
-/* GET current user's rating for snippet */
-router.get('/:snippet_id/user', function (req, res) {
-  var snippet_id = req.params.snippet_id;
-  if(req.user){
-    var user_id = req.user.dataValues.id;
-    models.Rating.findOne({ where : { SnippetId: snippet_id , UserId: user_id } }).then( function (rating) {
-      var grade = 0;
-      if (rating) {
-        grade = rating.value;
-      } 
-      var hash = {user: user_id, rate: grade, snippet: snippet_id};
-      res.status(201).send(hash);
-    });
-  } else {
-    res.status(422).send({user: null, rate: 0, snippet: snippet_id});
-  }
-});
-
-/* GET snippet's average rating */
-router.get('/:id/avg', function (req, res){
-  var snippet_id = req.params.id;
-  var sum_ratings = 0.0;
-  models.Rating.sum('value', { where : { SnippetId: snippet_id } }).then(function (sum){
-    sum_ratings = sum;
-    models.Rating.count({ where : { SnippetId: snippet_id } }).then(function (c){
-      var average = 0;
-      if(c > 0) {
-        average = (sum_ratings/c);
-      }
-      res.status(200).send({avg: average.toFixed(2), snippetId: snippet_id});
-    });
-  });
-});
 
 /* DELETE snippet and it's ratings/versions/comments */
 router.delete('/:id', function (req, res){
-  var snippet_id = req.params.id;
-  models.Snippet.findById(snippet_id).then(function (snippet){
+  var snippetId = req.params.id;
+  models.Snippet.findById(snippetId).then(function (snippet){
```

Comment:
it will work perfectly well but you can write it shorter in the future and probably a little more readable:
```
  models.Snippet.findById(snippetId).then(function (snippet){
      return snippet.destroy().then(function () {
        res.status(200).send({snippet: snippetId});
      });
   }).catch(function (err) {
     res.status(422).send(err);
   });
 });
```